Generals/Code/Libraries/Source/WWVegas/WW3D2/decalmsh.cpp

```diff
@@ -26,12 +26,13 @@
  *                                                                                             *
  *              Original Author:: Greg Hjelstrom                                               *
  *                                                                                             *
- *                      $Author:: Greg_h                                                      $*
+ *                      $Author:: Kenny Mitchell                                               * 
+ *                                                                                             * 
+ *                     $Modtime:: 06/26/02 4:04p                                             $*
  *                                                                                             *
- *                     $Modtime:: 7/26/01 9:03a                                               $*
- *                                                                                             *
- *                    $Revision:: 20                                                          $*
+ *                    $Revision:: 24                                                          $*
  *                                                                                             *
+ * 06/26/02 KM Matrix name change to avoid MAX conflicts                                       *
```

Comment:
Is there a possibility to use `Matrix3`, `Matrix4` names in Zero Hour?

Reply:
Well, the way EA had it is there's pluglib for tools, i don't think we should have that tho, pluglib has really old code which exemplifies why having a clone of the code isn't a good idea

Reply:
You mean we should not build the max2w3d plugin? Because of the name collision?

Reply:
I mean we should make it build off this code not pluglib

Reply:
I do not understand.